app/src/controllers/organizations/reducer.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { combineReducers } from 'redux';
+import {} from 'controllers/filter/constants';
+import {
+  ORGANIZATIONS_INITIAL_STATE,
+  FETCH_ORGANIZATIONS_SUCCESS,
+  FETCH_ORGANIZATIONS,
+} from './constants';
+
+export const organizationsInfoReducer = (
+  state = ORGANIZATIONS_INITIAL_STATE,
+  { type = '', payload = {} },
+) => {
+  switch (type) {
+    case FETCH_ORGANIZATIONS_SUCCESS:
+      return { ...state, ...payload };
+    default:
+      return state;
+  }
+};
+
+export const organizationsLoadingReducer = (state = false, { type = '' }) => {
+  switch (type) {
+    case FETCH_ORGANIZATIONS:
+      return true;
+    case FETCH_ORGANIZATIONS_SUCCESS:
+      return false;
+    default:
+      return state;
+  }
+};
+
+export const organizationsReducer = combineReducers({
+  info: organizationsInfoReducer,
```

Comment:
I suggest to name it as `list` instead of info to easier understand that it is an array.

Reply:
Done.